subsys/bluetooth/audio/bap_unicast_client.c

```diff
@@ -51,6 +50,7 @@ struct bt_bap_unicast_client_ep {
 
 	/* Bool to help handle different order of CP and ASE notification when releasing */
 	bool release_requested;
+	bool cp_ntf_pending;
```

Comment:
is this initialized somewhere?

Reply:
Nice catch - I had wrongly expected the `struct bt_bap_unicast_client_ep` to be memset on reset, but it only resets a few fields. 

Added initialization of `release_requested` and `cp_ntf_pending`